contracts/Opet_coin.sol

```diff
@@ -316,20 +238,30 @@ contract OpetToken is MintableToken {
     transferPaused = false;
   }
 
-  function transferFrom(address _from, address _to, uint256 _value) notPaused public returns (bool) {
+
+  function transferFrom(address _from, address _to, uint256 _value) tranferable public returns (bool) {
       return super.transferFrom(_from, _to, _value);
   }
 
-  function transfer(address _to, uint256 _value) notPaused public returns (bool) {
+  function transfer(address _to, uint256 _value) tranferable public returns (bool) {
     return super.transfer(_to, _value);
   }
 
-  function sendAirdrops(address[] _addresses, uint256[] _amounts) hasMintPermission canMint public {
+  function sendAirdrops(address[] _addresses, uint256[] _amounts) public {
```

Comment:
Do we need to mark this method as `onlyOwner`? As I understand, it is logically can only be used by owner who has all the tokens, right? Would be good to explicitly show this with the modifier.